cartridges/openshift-origin-cartridge-jbosseap/metadata/manifest.yml

```diff
@@ -68,7 +69,7 @@ Endpoints:
     Backend: ''
     Options:
       websocket: true
-  - Frontend: /health
+  - Frontend: "/health"
```

Comment:
i don't think these quotes matter either way, i'm not sure why the tool insists on adding them but it may be a path of least resistance to just leave them.